code/game/machinery/incident_display.dm

```diff
@@ -117,35 +146,22 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/incident_display/tram, 32)
 
 	if(sign_features == DISPLAY_TRAM)
 		tool.play_tool_sound(src)
-		balloon_alert(user, "set to dual")
-		name = NAME_DUAL
-		desc = DESC_DUAL
-		icon_state = "stat_display_dual"
-		sign_features = DISPLAY_DELAM | DISPLAY_TRAM
+		balloon_alert(user, "set to delam")
```

Comment:
Might be a bit out of scope, but does this interaction have an examine/tooltip yet so players know multitools can do this? Same question goes for the welder and repairs

Reply:
I'll double check. I think maybe? If not I'm fine with adding one.

Reply:
Done.